### PR TITLE
Create PKGBUILD

### PR DESCRIPTION
**Pull request by hollma**
Install omikhleia-sile-packages via pacman (Arch Linux)

### CODE REVIEW

**Comment by Omikhleia** [comment]
Greetings @hollma and thanks for the proposal.

I'm currently on a book project using my classes & packages - this also helps testing them in a larger case than the existing documentation - so I might delay merging this PR a bit, until I am sure the classes and packages work well on a "real" example (beyond the mere documentation).

I might also remove the "examples" from the packaging (first because they have varying licenses, but also because I might split them separately as they grow).

I might also try to transfer some of these packages to the SILE standard distribution, perhaps... Would they think them worth... That is to say, I am not sure yet when I'll address you PR and what it will encompass then exactly. But whatever stays here eventually, I do agree that a proper packaging is indeed needed.

**Comment by alerque** [comment]
As an Arch Linux user and developer myself this is interesting to see ... but I'm pretty sure this repository is not the right place for a PKGBUILD. Long term I'm pretty sure there should not be a need for this at all (when SILE's package manager works right and/or we use Luarocks for distributing packages like this, see #538), but in the short term I think using a PKGBUILD for this is perfectly valid. I would just suggest that this belongs in the AUR. Would you consider posting it there or should I? Or can we both co-maintain it?

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
+_appname=omikhleia-sile-packages
+_builddir=build
+
+pkgname=${_appname}-git
```
Comment (alerque) [inline PKGBUILD:4]: For consistency with Arch packaging guidelines and other future possible packages, when putting this on the AUR I'd suggest a name like this:

```suggestion
pkgname=sile-package-omikhleia-git
```

**Comment by alerque** [comment]
I have overhauled this to be a bit more in line with Arch packaging guidelines and posted it to the AUR as [sile-package-omikhleia-git](https://aur.archlinux.org/packages/sile-package-omikhleia-git). As I [commented on the related SILE issue](https://github.com/sile-typesetter/sile/issues/324#issuecomment-1011103512) this installation method is a total hack, but until we have something more appropriate I guess it will pass. I'd be happy to add either of you as co-maintainers or the AUR if you let me know your username or PRs are always accepted [on my AUR related repo](https://github.com/alerque/aur/tree/master/sile-package-omikhleia-git).

**Comment by Omikhleia** [comment]
Thanks for your interest, folks !
I know it has been a long time...

However, I am closing (= rejecting this PR) for several reasons:
- This is likely not the proper way for bundling packages and class for SILE, and the latter will have eventually its own clean way of implementing this (via luarocks or whatever package management is currently under consideration) 
- In the meantime, several packages from this repository have been proposed upstream to SILE.
- The target here in this repository is still on SILE 0.12.x, so upgrading this stuff would have to be considered for it to remain decently usable and future-proof again. AFAIC, it won't happen, almost likely (for those things that were not proposed upstream).
- I might even split this repository and move some of the things elsewhere.

**Pull request closed without merging**